app/core/steps/AddressStep.js

```diff
@@ -16,7 +17,8 @@ class AddressStep extends ValidationStep {
         return [ctx];
     }
 
-    handlePost(ctx, errors) {
+    handlePost(ctx, errors, formdata, session, hostname, featureToggles) {
```

Comment:
Can we have a unit test for at least the new variable that was added to this method